contracts/DssVestTopUp.sol

```diff
@@ -68,6 +79,17 @@ contract DssVestTopUp is IUpkeepRefunder, Ownable {
         uint256 _maxDepositAmt,
         uint256 _threshold
     ) {
+        require(_dssVest != address(0), "invalid dssVest address");
+        require(_daiJoin != address(0), "invalid daiJoin address");
+        require(_vow != address(0), "invalid vow address");
+        require(_paymentToken != address(0), "invalid paymentToken address");
+        require(_keeperRegistry != address(0), "invalid keeperRegistry address");
+        require(_swapRouter != address(0), "invalid swapRouter address");
+        require(_linkToken != address(0), "invalid linkToken address");
```

Comment:
Wouldn't it be better to generalize the require for non 0x0 address? Or do you want to keep the error messages more explicit?

Reply:
Not sure how to do it in a way that will actually simplify it without obscuring it for the sake of brevity. Open for suggestions.